slothy/targets/aarch64/cortex_a72_frontend.py

```diff
@@ -200,6 +201,7 @@ def get_min_max_objective(slothy):
         ExecutionUnit.ASIMD1,
     ],
     AArch64NeonShiftInsert: [ExecutionUnit.ASIMD1],
+    AArch64ConditionalCompare: [ExecutionUnit.INT()],
```

Comment:
Why does this occupy both integer pipelines? Shouldn't it be just one?
```suggestion
    AArch64ConditionalCompare: ExecutionUnit.INT(),
```

Reply:
So, if I model
```
AArch64ConditionalCompare: [ExecutionUnit.INT()],
```
then SLOTHY treats this as: AArch64ConditionalCompare occupy both I0 AND I1
 
if I model
```
AArch64ConditionalCompare: ExecutionUnit.INT(),
```
then AArch64ConditionalCompare occupy I0 OR I1

Reply:
Yes,  remember that ExecutionUnit.INT() resolves to `[ExecutionUnit.INT0, ExecutionUnit.INT1]` that means INT0 OR INT1
`[[ExecutionUnit.INT0, ExecutionUnit.INT1]]` means INT0 AND INT1.

I know this can be confusing and we may have gotten it wrong sometimes.

Reply:
Thanks!